octoversion.json

```
@@ -1,3 +1,3 @@
 {
-    "NonPreReleaseTags": ["refs/heads/main"]
+    "NonPreReleaseTags": ["main"]
```

Recommend supporting both - thats what we do elsewhere
```suggestion
    "NonPreReleaseTags": ["refs/heads/main", "main"]
```